src/main/java/com/github/manolo8/darkbot/gui/players/PlayerEditor.java

```diff
@@ -22,10 +22,10 @@
 import java.util.Set;
 
 public class PlayerEditor extends JPanel implements Listener {
-    private final JList<PlayerInfo> playerInfoList;
+    final JList<PlayerInfo> playerInfoList;
     private final DefaultListModel<PlayerInfo> playersModel;
     private final PlayerRenderer renderer;
-    private final Set<PlayerInfo> nearbyPlayers = new LinkedHashSet<>();
+    final Set<PlayerInfo> nearbyPlayers = new LinkedHashSet<>();
```

Comment:
```suggestion
    private final Set<PlayerInfo> nearbyPlayers = new LinkedHashSet<>();
```
Keep it private, if you need to access it add a getter